packages/video_player/video_player_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.0.0-nullsafety.1
+
+* Add note about the `mixWithOthers` option being ignored on the web.
+
```

Comment:
Ah, thanks, sorry about missing this one

Reply:
No problem! Thanks for the contribution! Keep them coming! :D